scripts/st2_bootstrap.sh

```diff
@@ -181,5 +187,7 @@ else
 
     echo "Running deployment script for st2 ${VERSION}..."
     echo "OS specific script cmd: bash ${BOOTSTRAP_FILE} ${VERSION} ${RELEASE} ${REPO_TYPE} ${USERNAME} ${PASSWORD}"
-    bash ${BOOTSTRAP_FILE} ${VERSION} ${RELEASE} ${REPO_TYPE} ${DEV_BUILD} ${USERNAME} ${PASSWORD}
+    TS=$(date +%Y%m%dT%H%M%S)
+    sudo mkdir -p /var/log/st2
+    bash ${BOOTSTRAP_FILE} ${VERSION} ${RELEASE} ${REPO_TYPE} ${DEV_BUILD} ${USERNAME} ${PASSWORD} 2>&1 | adddate | sudo tee /var/log/st2/st2-install.${TS}.log
```

Comment:
Nice trick :+1:

Reply:
Just please verify if entire script hangs with the correct exit code if something is broken during the main installation.

Reply:
Good catch. I'll submit a fix for that.

Reply:
I've verified `exit ${PIPESTATUS[0]}` works as expected.